databases/data_collection_database.py

```diff
@@ -33,6 +33,15 @@ def __init__(self, experiment: str = None, measurement_items: list = ["Weight",
         
     #     self.measurement_items = measurement_items
       
+        # Updated to ensure the file creation logic works if the file doesn’t exist
+        # path = Path(self.filename)
+        # if not path.is_file():
+        #     with open(self.filename, 'w') as f:
+        #         headers = "Date,Animal ID," + ",".join(self.measurement_items) + "\n"
+        #         f.write(headers)
+
+
+
```

Comment:
Is this necessary? Would this screen ever be accessed if the file is not already existing?

Reply:
```    #     if not path.is_file():
    #         f = open(self.filename, "x")
    #         string = "Date,Animal ID,"
    #         for item in measurement_items:
    #             string += item + ","
    #         string = string[:-1]
    #         f.write(string)
    #         f.close()```

This code is already commented on lines 25-32 and looks similar to this. If the commented code needs to be updated/deleted/uncommented you can use the pre-existing code.